### ISSUE TITLE
Configuration property error

### ISSUE THREAD

**Issue by darthmat** [php-rdkafka/php-rdkafka#426]
Hi!

After create basic configuration of php-kafka i wanted to use:
`$conf->set('ssl.endpoint.identification.algorithm', 'https');`

Based on [https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md)
such config exist.

Unfortunately when im sending such config on my gitlab branch i have some php unit tests error with:
`RdKafka\Exception: No such configuration property: "ssl.endpoint.identification.algorithm"`

my dockerfile contain this config so seems that all needed libs are installed:

```docker
FROM php:7.3.17-apache
WORKDIR /app
EXPOSE 80
RUN mv "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/php.ini"
RUN apt update && apt install -y libldap2-dev librdkafka-dev libzip-dev zip libssl-dev openssl
RUN docker-php-ext-install ldap pdo_mysql zip
RUN pecl install rdkafka && docker-php-ext-enable rdkafka
COPY --from=build-php /app /app
COPY --from=build-node /app/public /app/public
COPY vhost.conf /etc/apache2/sites-available/000-default.conf
COPY prod.env /app/.env
RUN chown -R www-data:www-data /app \
    && a2enmod rewrite
````

can someone help me with this error? Or why this error occure?

**Comment by nick-zh** [comment]
What version of librdkafka are you using?
Also make sure` OpenSSL >= 1.0.2` as stated in `librdkafka`

**Comment by darthmat** [comment]
`librdkafka-dev:amd64 (0.11.6-1.1)` and `openssl (1.1.1d-0+deb10u4)`

seems that versions are correct?

**Comment by nick-zh** [comment]
`0.11.6` is quite old, can you try if you have the same for a newer version?

**Comment by darthmat** [comment]
hmmm seems that for some reason `apt install librdkafka-dev=1.5.0` not works (i get "there is no such version"). Maybe im doing something wrong?

But, cloning repo and building this lib from scratch with this, works and error is gone:
```
ENV LIBRDKAFKA_VERSION v1.5.0
ENV BUILD_DEPS \
        build-essential \
        git \
        libsasl2-dev \
        libssl-dev \
        python-minimal \
        zlib1g-dev
RUN apt-get update \
    && apt-get install -y --no-install-recommends ${BUILD_DEPS} \
    && cd /tmp \
    && git clone \
        --branch ${LIBRDKAFKA_VERSION} \
        --depth 1 \
        https://github.com/edenhill/librdkafka.git \
    && cd librdkafka \
    && ./configure \
    && make \
    && make install \
    && pecl install rdkafka \
    && docker-php-ext-enable rdkafka \
    && rm -rf /tmp/librdkafka \
    && apt-get purge \
        -y --auto-remove \
        -o APT::AutoRemove::RecommendsImportant=false \
        ${BUILD_DEPS}
```

its adding some more steps unfortunately. Do you know why `apt install librdkafka-dev=1.5.0` return "no such version"?

**Comment by nick-zh** [comment]
Yes because `debian:buster` (and a lot of other distros for that fact) come with a very old `librdkafka` version. In buster the only available version is `librdkafka:0.11.6`
You need to install it manually (as you did above) or alternatively you can switch to an alpine image which has more up to date librdkafka versions available.

**Comment by darthmat** [comment]
Great, thanks for help!
